Source/WebCore/page/InteractionRegion.h

```diff
@@ -48,6 +48,7 @@ struct InteractionRegion {
     ElementIdentifier elementIdentifier;
     Region regionInLayerCoordinates;
     float borderRadius { 0 };
+    bool isOccluding { false };
```

Comment:
This should probably be an `enum class X : bool` so we can have good names at initialization sites instead of random true/false in the last position